### DISCUSSION TITLE
Forward some messages based on condition to another SMTP server

### DISCUSSION THREAD

**Discussion by tiredofit** [stalwartlabs/smtp-server#34] [Q&A]
I am trying to match my postfix configuration, which allows for adding to `transport_maps` a file with regular expressions to allow for routing to other domains. My use case is that hotmail|msn|live|outlook domains are very strict for filtering, and I use a third party SMTP to deliver those messages like so:

```
/.*@hotmail.*/i             smtp:[smtp.secondarymta.com]:587
/.*@outlook.*/i             smtp:[smtp.secondarymta.com]:587
/.*@live.*/i                smtp:[smtp.secondarymta.com]:587
/.*@msn.*/i                 smtp:[smtp.secondarymta.com]:587
```

I provide authentication to that secondary mail server via `smtp_sasl_password_maps` like so:

`[smtp.secondarymta.com]:587 smtp_user:super_secure_password`

Any advice on replicating would be welcomed

**Comment by mdecimus** [comment]
Yes, this can be done using SMTP routing rules. Check the routing section on the documentation for examples.
